DESCRIPTION

```diff
@@ -33,6 +33,7 @@ Imports:
     lifecycle,
     R6,
     rlang (>= 1.1.0),
+    roxygen2,
```

Comment:
I'd have preferred to make this an optional dependency but I don't know how to not have the imports like roxy_tag_parse in DESCRIPTION.

Should the tag/roclet actually live in yet another package? So `use_vcr_examples()` would set up a dependency on that package, called vcr.roxygen2 or something like that?